pkg/replication/replicator.go

```diff
@@ -143,10 +143,13 @@ func (r replicator) initializeReplicationWithCheck(
 		errCh:     errCh,
 		replicationCancelledCh: make(chan struct{}),
 		replicationDoneCh:      make(chan struct{}),
+		enactedCh:              make(chan struct{}),
 		quitCh:                 make(chan struct{}),
 		concurrentRealityRequests: make(chan struct{}, concurrentRealityRequests),
 		timeout:                   r.timeout,
 	}
+	// To make a closed channel
+	close(replication.enactedCh)
```

Comment:
It's made so that if someone tries to call `WaitForReplication()` they wouldn't block if they didn't call enact.